### PR TITLE
MARP-1308 Update name and change field type

### PR DESCRIPTION
**Pull request by ndkhanh-axonivy**

### CODE REVIEW

**Comment by ndkhanh-axonivy** [comment]
Hi @ivy-fhe, could you please help me to check this PR? I just modify small stuffs such as refactoring the property name and its type. Thank you!

File: src/process-mining-visualisation/mining-action.ts [hunk 1]
```diff
@@ -10,18 +10,18 @@ export interface MiningAction extends Action {
...
 }
 
 export interface MiningNode {
   type: string;
-  ID: string;
-  relativevalue: number;
-  labelvalue: string;
+  id: string;
+  relativeValue: number;
+  labelValue: number;
```
Comment (ndkhanh-axonivy) [inline src/process-mining-visualisation/mining-action.ts:24]: We change this type due to the requirement from story: https://1ivy.atlassian.net/browse/MARP-974

Comment (ivy-fhe) [reply src/process-mining-visualisation/mining-action.ts:24]: Looks fine to me :smile:

**Comment by ndkhanh-axonivy** [comment]
Hi @ivy-fhe , I just add a commit for handle conflict. Could you help me to check it again, thank you!

**Review by ivy-fhe — APPROVED** [review]
:+1:

**Pull request merged**